DataProcessing/datax-core/src/main/scala/datax/constants/BlobProperties.scala

```diff
@@ -7,4 +7,6 @@ package datax.constants
 object BlobProperties {
   // Define constants for blobs
   val BlobHostPath = ".blob.core.windows.net"
+  val AzureStorageJarPath = "/datax/bin/azure-storage-3.1.0.jar"
+  val DefaultContainer = "defaultdx"
```

Comment:
If we have an external config, let's move this as well.

Reply:
moved as environment variable